src/main/scala/castalia/actors/JsonEndpointActor.scala

```diff
@@ -24,40 +22,30 @@ class JsonEndpointActor( myStubConfig: StubConfig) extends Actor with ActorLoggi
 
   override def receive: Receive = {
     case request: RequestMatch =>
-      log.debug("receive requestmatch")
```

Comment:
Why remove all these debug statements? when in production, log level should be >=Info.